panel/1.0/troubleshooting.md

```diff
@@ -140,7 +140,7 @@ If this returns different DNS Servers than 1.1.1.1 and 1.0.0.1 you'll need to ed
 - Check logs from your queue manager ``journalctl -xeu pteroq``
 - Restart pteroq ``systemctl restart pteroq``
 - Clear schedule cache ``php /var/www/pterodactyl/artisan schedule:clear-cache``
-- Check your php version - up to 8.3 is supported ``php -v``
+- Check your php version ``php -v`` - [this page](<https://pterodactyl.io/panel/1.0/updating.html#panel-version-requirements>) will tell you what versions of php are supported by what versions of the panel
```

Comment:
Line 143 is incorrect, the `<>` are not needed in the context you are using and should be removed. 

The `<>` are used independently when you want to pop something out which is just a link, not an embedded link, such as in the contab guru link below. 

Example: 
```
[this page](https://pterodactyl.io/panel/1.0/updating.html#panel-version-requirements)
```

Thanks for your PR!